apps/files/src/views/FilesList.vue

```diff
@@ -294,6 +295,16 @@ export default defineComponent({
 				return []
 			}
 
+			let filteredDirContent = [...this.dirContents]
+			// Filter based on the provided filterText
+			if (this.filterText) {
+				filteredDirContent = filteredDirContent.filter(content => {
+					// Adjust this condition based on your specific filtering criteria
+					return content.attributes[this.sortingMode].toLowerCase().includes(this.filterText.toLowerCase())
```

Comment:
Why filter by attribute? :thinking: 
```suggestion
					return content.basename.toLowerCase().includes(this.filterText.toLowerCase())
```